api/src/main/java/com/inrupt/client/Response.java

```diff
@@ -97,6 +95,15 @@ interface ResponseInfo {
         ByteBuffer body();
     }
 
+    /**
+     * Indicates whether a status code reflects a successful HTTP response.
+     * @param statusCode the HTTP response status code
+     * @return true if the status code is in the success range, namely [200, 299].
+     */
+    static boolean isSuccess(final int statusCode) {
```

Comment:
Do we really need this method?

Reply:
It avoids some duplication, and it is a quite common pattern to have this affordance on HTTP response object: JS `Response` has a `.ok` getter, `OkHttp` has [`isSuccessful`](https://square.github.io/okhttp/5.x/okhttp/okhttp3/-response/is-successful.html)... I'm not strongly opposed to removing this, but I find it reads slightly better than checking the status range each time.